challenge1.js

```diff
@@ -0,0 +1,15 @@
+const vowelCounter = ( string ) => {
+
+    let vowelCounter = 0;
+    for ( let letter of string ) {
+        if ( letter === `a` || letter === `e` || letter === `i` || letter === `o` || letter === `u` ) {
+            vowelCounter += 1;
+        }
+    }
```

Comment:
Something like this is neater and more extensible
```suggestion
    const vowels = [`a`, `e`, `i`, `o`, `u`];

    let vowelCounter = 0;
    for (let i = 0; i < string.length; i++) {
      if (vowels.includes(string[i])) {
        vowelCounter++;
      }
    }
```